samples/SocialSample/Startup.cs

```diff
@@ -218,7 +215,7 @@ 127.0.0.1 MsSecSample.localhost.this
             // Deny anonymous request beyond this point.
             app.Use(async (context, next) =>
             {
-                if (string.IsNullOrEmpty(context.User.Identity.Name))
+                if (context.User == null || !context.User.Identities.Any(identity => identity.IsAuthenticated))
```

Comment:
Unrelated change, but I fixed it because the "GitHub-AccessToken" sample didn't work properly.